### PR TITLE
Add readme for FE

### PR DESCRIPTION
**Pull request by bluebarryz**

### CODE REVIEW

**Review by max-y-huang — APPROVED** [review]

File: fe/README.md [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+
+Inside `/fe`, create a directory called `/assets`, if it does not already exist. 
+
+Add `happy_birthday.mxl` to this `/assets` directory, if it does not already exist.
+> You can also upload any other `.mxl` file of your choosing, just make sure to change the `filename` in `main.dart` to be the name of your file:
+```
+const filename = "happy_birthday.mxl";
+```
+
+Select the Web as your device and run the Flutter app!
```
Comment (ngtr6788) [inline fe/README.md:27]: You can change this to something akin to:

To run this Flutter app, either connect to an iPad or Android tablet device, or start up a tablet emulator on your development machine, then run `flutter run`.

File: fe/README.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
-## Getting Started
+## Install backend dependencies
+```
+cd be
+npm install --immutable
+```
 
-This project is a starting point for a Flutter application.
+## Run the server
+Inside `/be`, create a `.env` file with the contents `NODE_PORT=3000`. Now we can run the server at port 3000:
```
Comment (ngtr6788) [inline fe/README.md:11]: You can write `NODE_PORT=<your port here>`

File: fe/README.md [hunk 3]
```diff
@@ -1,16 +1,30 @@
...
+```
+npm run dev
```
Comment (ngtr6788) [inline fe/README.md:13]: Also add something like "Optionally, you can also run `npm run dev:tunnel` to start the server as well as a tunnel created by [localhost.run](https://localhost.run/)

File: fe/README.md [hunk 4]
```diff
@@ -1,16 +1,30 @@
...
+```
+
+## Run the Flutter app on the Web (Chrome)
+Open a new terminal window and navigate to the `/fe` directory.
+
+Inside `/fe`, create a directory called `/assets`, if it does not already exist. 
+
+Add `happy_birthday.mxl` to this `/assets` directory, if it does not already exist.
+> You can also upload any other `.mxl` file of your choosing, just make sure to change the `filename` in `main.dart` to be the name of your file:
```
Comment (ngtr6788) [inline fe/README.md:22]: as well as adding that file to the `assets` section in `pubspec.yaml`

**Review by ngtr6788 — COMMENTED** [review]
I made some suggestions to the README. Feel free to come up with something better

File: fe/README.md (hunk 2 above)
Comment (max-y-huang) [reply fe/README.md:11]: Same with `PYTHON_PORT` in the future

Comment (bluebarryz) [reply fe/README.md:11]: Sounds good

**Pull request closed without merging**